apollo-router-core/src/query_cache.rs

```diff
@@ -40,8 +40,8 @@ impl QueryCache {
     }
 
     /// Attempt to parse a string to a [`Query`] using cache if possible.
-    #[tracing::instrument(skip_all, level = "info" name = "get_query")]
-    pub async fn get_query(&self, query: impl AsRef<str>) -> Option<Arc<Query>> {
+    #[tracing::instrument(skip_all, level = "info" name = "parse_query")]
```

Comment:
Is this comment (and the associated re-name) correct? I can't see any parsing, it's just trying to `get` the query (`Query`) using a key of a `String`.

Reply:
I've put the method name back, and moved the instrumentation to the actual point where the query gets parsed. This means that query parsing will generate a span, but cache hits won't.